### PR TITLE
Some improvements for gettext-translations

### PR DESCRIPTION
**Pull request by besser82**

### CODE REVIEW

File: dnfdragora/progress_ui.py [hunk 1]
```diff
@@ -175,7 +175,8 @@ def get_total(self):
 
     def update(self):
         """ Output the current progress"""
-        text = _("Progress files (%d/%d)") % (self.download_files, self.total_files)
+        text = _("Progress files (%(downloaded)d/%(total)d)") %
+            {'downloaded': self.download_files, 'total': self.total_files}
```
Comment (Conan-Kudo) [inline dnfdragora/progress_ui.py]: Invalid syntax here.

Comment (besser82) [inline dnfdragora/progress_ui.py]: Fixed now

File: dnfdragora/dnf_backend.py [hunk 2]
```diff
@@ -242,8 +242,8 @@ def on_DownloadStart(self, num_files, num_bytes):
         self._files_downloaded = 0
         self.frontend.infobar.set_progress(0.0)
         self.frontend.infobar.info_sub(
-            _('Downloading %d files (%sB)...') %
-            (num_files, dnfdragora.misc.format_number(num_bytes)))
+            _('Downloading %(num_files)d files (%(format_num)sB)...') %
+            {'num_files':num_files, 'foramt_num': dnfdragora.misc.format_number(num_bytes)})
```
Comment (Conan-Kudo) [inline dnfdragora/dnf_backend.py]: I get a traceback on Mageia Cauldron with this during installing packages...

```
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/dnfdaemon/client/__init__.py", line 187, in __call__
    return getattr(self.proxy, self.method)(*args)
  File "/usr/lib/python3.5/site-packages/dnfdaemon/client/__init__.py", line 228, in _on_g_signal
    self.handle_dbus_signals(proxy, sender, signal, args)
  File "/usr/lib/python3.5/site-packages/dnfdaemon/client/__init__.py", line 560, in handle_dbus_signals
    self.on_DownloadStart(*args)
  File "/home/ngompa/Projects/b82-dnfdragora/dnfdragora/dnf_backend.py", line 246, in on_DownloadStart
    {'num_files':num_files, 'foramt_num': dnfdragora.misc.format_number(num_bytes)})
KeyError: 'format_num'
```

Comment (besser82) [inline dnfdragora/dnf_backend.py]: It is due a slight typo…  Fixed…

**Comment by Conan-Kudo** [comment]
Looks good to me now.

**Pull request merged**